automations/js/src/utils/projects.mjs

```diff
@@ -27,11 +27,13 @@ class Project {
    *                         owner              number
    *
    * @param octokit {import('@octokit/rest').Octokit} the Octokit instance to use
+   * @param core {import('@actions/core')} GitHub Actions toolkit, for logging
    * @param owner {string} the login of the owner (org) of the project
    * @param number {number} the number of the project
    */
-  constructor(octokit, owner, number) {
+  constructor(octokit, core, owner, number) {
     this.octokit = octokit
+    this.core = core
```

Comment:
Instead of passing `core` individually to functions inside `Project`, it is cleaner to pass it once, during `Project` construction.